#
#

Header header
BatteryStatus batteryStatus
EStopStatus eStopStatus
SonicImpact sonicImpact
uint32 upTime
uint32 errorCode
bool brakeStatus
float32 temperature
bool cargoLock
uint16 cargoLid
bool manualMode
